### PR TITLE
Add performance test for 100% write inserts scenario in write-back mode.

### PR DESCRIPTION
**Pull request by KlaudiaJ**
Signed-off-by: Klaudia Jablonska <klaudia.jablonska@intel.com>

### CODE REVIEW

File: test/functional/tests/performance/test_performance_writes.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        core_device.create_partitions([data_size])
+        core_part = core_device.partitions[0]
+
+    with TestRun.step("Measure 4KiB write performance (throughput and latency) on raw disk."):
+        fio_command.target(cache_part)
+        for nj in num_jobs:
+            # jobs directed to specific offsets
+            offset = data_size.value // nj
+            # round down offset to multiplication of 512 blocks
+            offset = Size(offset // Unit.Blocks512.get_value() * Unit.Blocks512.get_value())
```
Comment (Deixx) [inline test/functional/tests/performance/test_performance_writes.py:76]: I'm not sure, but do we still support 512B? Should it be 4KiB instead?

File: test/functional/tests/performance/test_performance_writes.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            TestRun.LOGGER.info(f"Measuring performance for num_jobs={nj}")
+
+            with TestRun.step("Start cache and add core device"):
+                cache = casadm.start_cache(cache_part, CacheMode.WB, cache_line_size, cache_id=1)
+                cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+                core = cache.add_core(core_part, core_id=1)
+
+            with TestRun.step("Ensure that I/O scheduler for CAS device is 'none'"):
+                TestRun.executor.run_expect_success(
+                    "sudo echo none > /sys/block/cas1-1/queue/scheduler"
```
Comment (Deixx) [inline test/functional/tests/performance/test_performance_writes.py:100]: hardcoded cas1-1 (I know IDs are set explicitly, but still)

File: test/functional/tests/performance/test_performance_writes.py (hunk 1 above)
Comment (katlapinka) [reply test/functional/tests/performance/test_performance_writes.py:76]: both are still supported :)

File: test/functional/tests/performance/test_performance_writes.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                )
+
+            with TestRun.step("Run fio on CAS device"):
+                fio_command.target(core)
+                # jobs directed to specific offsets
+                offset = data_size.value // nj
+                # round down offset to multiplication of 512 blocks
+                offset = Size(offset // Unit.Blocks512.get_value() * Unit.Blocks512.get_value())
```
[the remark this answers is not in the document]
Comment (katlapinka) [reply test/functional/tests/performance/test_performance_writes.py:108]: 512B is still supported

**Comment by katlapinka** [comment]
Moved to test/functional/tests/performance/test_performance_write.py

**Pull request closed without merging**